packages/next/build/index.ts

```diff
@@ -1,3 +1,4 @@
+require('module-alias/register')
```

Comment:
Add use the advanced usage: https://www.npmjs.com/package/module-alias#advanced-usage